configs/proxy_connect_udp.yaml

```diff
@@ -0,0 +1,59 @@
+# This will proxy CONNECT-UDP requests from a downstream connecting on 127.0.0.1:10001 to an
```

Comment:
Should we rename "proxy" to "forward" here to match [the spec](https://www.rfc-editor.org/rfc/rfc9298#section-3.1)?